cst/plugins/eu.esdihumboldt.cst.functions.geometric/help/extent/help.xhtml

```diff
@@ -8,23 +8,23 @@
 <div id ="help">
 	<h4>General explanation</h4>
 	<p>
-The Compute Extent function uses the source data geometries to derive the geometric extent and map it to the target geometry.
+The compute extent function uses the source data geometries to derive the geometric extent and map it to the target geometry.
 Three different options are available for extent computation: Bounding Box, Convex Hull and Union. Visual examples are 
 provided below.<br> 
 </br>
 <br> 
 </br>
 <strong>Bounding Box</strong><br>
  </br>
-The bounding box option maps the smallest rectangle shape that enclose the source data geometry to the target data geometry. <br>
+The bounding box option maps the smallest rectangle shape that encloses the source data geometries to the target data geometry. <br>
 </br>
 		<img src="../../eu.esdihumboldt.cst.functions.geometric/help/extent/ComputeExtent_BoundingBox.png" />
 
 <br> 
 </br>
 <strong>Convex Hull</strong><br>
  </br>
-The Convex Hull option maps the smallest convex shape that encloses the source data geometry to the target data geometry. <br>
+The Convex Hull option maps the smallest convex shape that encloses the source data geometries to the target data geometry. <br>
```

Comment:
"Convex Hull" --> "convex hull" if we decide to go with lower case for function/option names